zephyr/CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ set(SOF_TRACE_PATH "${SOF_SRC_PATH}/trace")
 
 # Save path to rimage configuration files in cmake cache for later use by
 # rimage during the "west sign" stage
-get_filename_component(RIMAGE_CONFIG "ext/rimage/config" ABSOLUTE)
+get_filename_component(RIMAGE_CONFIG "../rimage/config" ABSOLUTE)
```

Comment:
maybe we should remove this cached rimage path completely now and only use `-D`

Reply:
yeah, we could

Reply:
on a second thought, I do not think there is a harm, this is just a variable being set. If we remove it now, we will have to also remove it from the signing script, we can do this later.